docs/CONTRIBUTING.md

```diff
@@ -56,18 +56,41 @@ Vector team member will find this document useful.
 
 ### New sources, sinks, and transforms
 
-If you're contributing a new source, sink, or transform to Vector, thank you that's way cool! There's a few steps you
-need to think about if you want to make sure we can merge your contribution. We're here to help you along with these steps,
-but they are a blocker to getting a new integration released.
-
-To merge a new source, sink, or transform, you need to:
+If you're thinking of contributing a new source, sink, or transform to Vector, thank you that's way cool! There's a few
+steps you need to think about if you want to make sure we can merge your contribution. We're here to help you along with
+these steps, but they are a blocker to getting a new integration released.
```

Comment:
nit- the usage of the word "blocker" seems a little harsh. 
Suggested change to this last sentence:

"The answers to the below questions are required for each newly proposed component and depending on the answers, we may elect to not include the proposed component. If you're having trouble with any of the questions, we're available to help you."

?

Reply:
Technically those were referring to the old section of tests/integration/docs/etc - I could split it up so those are still referred to as blockers?

Reply:
Ah, I think the line on 86 could have that blocker word yeah. I do think it's worth pointing out that the new Q&A part is a requirement, independent of the tests/int/docs etc.